docs/sidebar.yml

```diff
@@ -123,7 +123,6 @@ subsection:
           - page: reference/dropped-features/type-projection.md
           - page: reference/dropped-features/do-while.md
           - page: reference/dropped-features/procedure-syntax.md
-          - page: reference/dropped-features/package-objects.md
```

Comment:
This page will still show up in google search results, so eventually it will need to have a comment mentioning it's outdated pointing to the new doc.